bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss

```diff
@@ -23,10 +27,30 @@
   flex: 0;
 }
 
-.center {
+.center,
+.nonVoicePresenter,
+.voicePresenter,
+.nonVoiceUser,
+.voiceUser {
   flex: 1;
 }
 
+.nonVoicePresenter {
+  margin-left: -11.6em;
```

Comment:
and what happen if a new button is added? Or the button size change? We should avoid fixed values, adding a new button shouldn't break the layout for example.